docs/.vitepress/theme/style.css

```
@@ -1 +1,142 @@
+  --vp-c-bg-soft: #f5f6f9;
+  --vp-c-divider: #e1e4e9;
+  --vp-c-text-1: #111827;
+  --vp-c-text-2: #5e6675;
+  --vp-button-alt-border: #c8cdd6;
+  --vp-button-alt-text: #111827;
+  --vp-button-alt-bg: #ffffff;
+  --vp-button-alt-hover-border: #9ca3af;
+  --vp-button-alt-hover-text: #111827;
+  --vp-button-alt-hover-bg: #f5f6f9;
+  --vp-font-family-base: ui-sans-serif, system-ui, -apple-system, "Segoe UI", sans-serif;
+  --vp-font-family-mono: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)